### DISCUSSION TITLE
Replace Chalk by Picocolors in dependency

### DISCUSSION THREAD

**Discussion by torresgol10** [vercel/turborepo#8341] [Ideas]
### Goals

Picocolors is a much smaller, better optimised and more efficient library.

### Background

Reducing the weight of dependencies and optimising ops/s

### Proposal

Yes, I could make the change of library.

**Comment by anthonyshew** [comment]
Sounds good to me! Those of us on the core team may not be able to find time to make these changes soon but we'd be happy to review a PR.

**Comment by torresgol10** [comment]
Hi @anthonyshew ,

I think this is not the best place, but I do not know whether to open another discussion, if you tell me what is necessary I can do, it is just a query.

Reviewing the turbo code, I have seen that in the package.json of the root contains `"node": "20.x"` and I have seen that in some packages the `rimraf` dependency is used minimally, do you think that we could replace it by the native one of node: https://nodejs.org/docs/latest-v20.x/api/fs.html#fsrmdirsyncpath-options

I saw that the parameters used with `rimraf` are also supported by the node version and this would remove a production dependency.

If you find it interesting, I can implement it

**Reply by anthonyshew** [reply]
Looks like you've got it in #8729! For the future, we're happy to see dependency upgrading PRs as long as they're sensible enough to be reviewed safely. 😄

**Reply by torresgol10** [reply]
OK, thank you very much.
